local ReplicatedStorage = game:GetService("ReplicatedStorage")

ReplicatedStorage:WaitForChild("Remotes")

local Controllers = script.Parent:WaitForChild("Controllers")
local AbilityHUDController = require(Controllers.AbilityHUDController)
local DashController = require(Controllers.DashController)
local UIController = require(Controllers.UIController)
local KillFeedController = require(Controllers.KillFeedController)
local KillConfirmController = require(Controllers.KillConfirmController)
local RoleRevealController = require(Controllers.RoleRevealController)
local FireAttackController = require(Controllers.FireAttackController)
local ViewmodelController = require(Controllers.ViewmodelController)
local CombatFeedbackController = require(Controllers.CombatFeedbackController)
local ChangeIndicatorController = require(Controllers.ChangeIndicatorController)

AbilityHUDController.init()
DashController.init()
UIController.init()
KillFeedController.init()
KillConfirmController.init()
RoleRevealController.init()
FireAttackController.init()
ViewmodelController.init()
CombatFeedbackController.init()
ChangeIndicatorController.init()
